Gemfile

```diff
@@ -1,6 +1,6 @@
 source :gemcutter
 
-gem 'nokogiri', "~>1.5.5"
+gem 'nokogiri', ">= 1.5.5"
```

Comment:
I understand the twiddle-waka better now, and I see what you did there. I'll rebase, change the code, and update the pull request at some point.